### PR TITLE
UI: Changed default page size for tables from 10 to 15

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on following - 
- Default page size for tables changed from 10 to 15
- Code optimisation  and bug fixes

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhookTable.tsx [hunk 1]
```diff
@@ -126,12 +126,15 @@ const WebhookTable: FC<Props> = ({ onEdit, webhookList, onDelete }) => {
...
-        ),
+        render: (_, record) =>
+          record.description ? (
+            <RichTextEditorPreviewer
+              markdown={record?.description || ''}
+              maxLength={100}
+            />
+          ) : (
+            <>--</>
+          ),
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhookTable.tsx:137]:
```suggestion
        render: (value) =>
          value ? (
            <RichTextEditorPreviewer
              markdown={value || ''}
              maxLength={100}
            />
          ) : (
            value
          ),
```

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/Webhooks/WebhookTable.tsx]: Done

File: openmetadata-ui/src/main/resources/ui/src/constants/constants.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const INITIAL_PAGING_VALUE = 1;
 export const JSON_TAB_SIZE = 2;
 export const PAGE_SIZE = 10;
 export const PAGE_SIZE_BASE = 12;
-export const PAGE_SIZE_MEDIUM = 16;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/constants/constants.ts:36]: what will happen if we are using count 16 somewhere?  did you check that scenario?

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

File: openmetadata-ui/src/main/resources/ui/src/constants/constants.ts (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/constants/constants.ts:36]: It was used in 2 places where we were also intending to change the size.

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**